www/index.html

```diff
@@ -29,9 +29,9 @@
 
         <div class="row">
             <div class="about-chabble col-xs-12 collapse" id="about-chabble">
-                <h4>Welcome to chatjs</h4>
+                <h4>Welcome to Chabble</h4>
```

Comment:
well personally I think capital letters are dumb. But I have no strong preferences. I don't think history is relevant here as we haven't really decided.